index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Aniket singh" />
+    <meta name="description" content="PlayX-shop e-commerce app for gaming devices." />
+    <link rel="stylesheet" href="https://playx-ui.netlify.app/styles/main.css" />
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/fa50d5d333.js" crossorigin="anonymous"></script>
```

Comment:
Font awesome can be used by link tags. Script is not needed. Please once verify it.

Reply:
change it